src/Bridge/Doctrine/Repositories/Lifecycle/ResponseRepository.php

```diff
@@ -0,0 +1,11 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\Webhooks\Bridge\Doctrine\Repositories\Lifecycle;
+
+use EoneoPay\Webhooks\Bridge\Doctrine\Repositories\FillableRepository;
+use EoneoPay\Webhooks\Bridge\Doctrine\Repositories\Interfaces\WebhookResponseRepositoryInterface;
+
+class ResponseRepository extends FillableRepository implements WebhookResponseRepositoryInterface
```

Comment:
Is it ok to have an empty repository extending `FillableRepository`?

Reply:
I've hit a similar situation a few times, and generally I don't go ahead with keeping it because I think to myself that someone will implement it if they really need it. I don't think it hurts to have this here, though.... do you think there will be any future methods implemented specific to this repository?

Reply:
The reason I created this empty repository because it felt weird to use `FillableRepository` in `Response` lifecycle entity.

Reply:
This approach is good: it means that we can have final or abstract classes only (this one should be final)

Reply:
**Resolution:**

`final` it is.